
From: Arnd Bergmann <arnd@arndb.de>

The watchdog ioctl interface is defined correctly for 32 bit emulation,
although WIOC_GETSUPPORT was not marked as such, for an unclear reason. 
WDIOC_SETTIMEOUT and WDIOC_GETTIMEOUT were added in may 2002 to the code
but never to the ioctl list.  This adds all three definitions.

Signed-off-by: Arnd Bergmann <arnd@arndb.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/compat_ioctl.h |    4 +++-
 1 files changed, 3 insertions(+), 1 deletion(-)

diff -puN include/linux/compat_ioctl.h~add-missing-watchdog-compatible_ioctls include/linux/compat_ioctl.h
--- 25/include/linux/compat_ioctl.h~add-missing-watchdog-compatible_ioctls	2004-08-04 20:41:08.464157048 -0700
+++ 25-akpm/include/linux/compat_ioctl.h	2004-08-04 20:41:08.469156288 -0700
@@ -595,12 +595,14 @@ COMPATIBLE_IOCTL(ATMTCP_REMOVE)
 COMPATIBLE_IOCTL(ATMMPC_CTRL)
 COMPATIBLE_IOCTL(ATMMPC_DATA)
 /* Big W */
-/* WIOC_GETSUPPORT not yet implemented -E */
+COMPATIBLE_IOCTL(WDIOC_GETSUPPORT)
 COMPATIBLE_IOCTL(WDIOC_GETSTATUS)
 COMPATIBLE_IOCTL(WDIOC_GETBOOTSTATUS)
 COMPATIBLE_IOCTL(WDIOC_GETTEMP)
 COMPATIBLE_IOCTL(WDIOC_SETOPTIONS)
 COMPATIBLE_IOCTL(WDIOC_KEEPALIVE)
+COMPATIBLE_IOCTL(WDIOC_SETTIMEOUT)
+COMPATIBLE_IOCTL(WDIOC_GETTIMEOUT)
 /* Big R */
 COMPATIBLE_IOCTL(RNDGETENTCNT)
 COMPATIBLE_IOCTL(RNDADDTOENTCNT)
_
